pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "kafe2"
+description = "Karlsruhe Fit Environment 2: a package for fitting and elementary data analysis"
+authors = [
+	{ name="Daniel Savoiu", email="daniel.savoiu@cern.ch" },
+	{ name="Johannes Gäßler", email="johannesg@5d6.de" },
+	{ name="Cedric Verstege", email="cedric.verstege@cern.ch" },
+	{ name="Günter Quast", email="G.Quast@kit.edu" },
+]
+readme = "README.rst"
+requires-python= ">=3.6"
+keywords = ["kafe2", "kit", "karlsruhe", "data", "analysis", "lab", "laboratory", "practical courses", "education", "university", "students", "physics", "fitting", "minimization", "minimisation", "regression", "parametric", "parameter", "estimation", "optimization", "optimisation"]
+
+license = {text = "GPL3"}
+classifiers = [
+        "Development Status :: 5 - Production/Stable",
+        "Intended Audience :: Education",
+        "Intended Audience :: Science/Research",
+        "Topic :: Education",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Scientific/Engineering :: Physics",
+        "Topic :: Scientific/Engineering :: Information Analysis",
+        "Topic :: Scientific/Engineering :: Visualization",
+        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+]
+#TODO dependency versions
+dependencies = [
+        "iminuit",
+        "NumPy",
+        "Numdifftools",
+        "Scipy",
+        "tabulate",
+        "matplotlib",
+        "PyYaml",
+        "six",
+        "funcsigs",
+        "SymPy",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+dev = [
+    "pylint ~=2.14.0",
+    "toml ~=0.10.2",
+    "yapf ~=0.32.0",
+]
```

Comment:
My previous comment was just some boilerplate. The actual dev dependencies can be found in https://github.com/PhiLFitters/kafe2/blob/master/dev_dependencies.txt
Sorry for the confusion